mii/config.py

```diff
@@ -49,6 +49,7 @@ class MIIConfig(BaseModel):
     max_tokens: int = 1024
     enable_load_balancing: bool = False
     replica_num: int = 1
+    restful_api_port: int = 0
```

Comment:
This should probably be a reasonable default value. Port 0 would be the wildcard port

Reply:
Also, if `load_balancing` has to be enabled for the restful api, would it make sense to automatically enable it when the restful api is enabled?

Reply:
Okay, I will add `enable_restful_api` to MIIConfig and set a reasonable number (e.g. 51080) as the default port.
I will also automatically enable `load_balancing` when `enable_restful_api` is True.